spec/models/beta/search/open_search_result_spec.rb

```diff
@@ -266,6 +266,20 @@
     it { expect { result.redirect! }.to change(result, :redirect?).from(nil).to(true) }
   end
 
+  describe '#intercept_message' do
+    subject(:result) { build(:search_result) }
+
+    it 'will match the object in the yaml file' do
+      searched_term = result.search_query_parser_result.corrected_search_query
+      expect(searched_term).to eq((I18n.t "#{searched_term}.title"))
```

Comment:
```suggestion
      expect(searched_term).to eq(I18n.t "#{searched_term}.title")
```

Reply:
Yea, i didn't want to add these braces too but apparently rubocop picks it up as an offence so that's why i added them.
<img width="586" alt="image" src="https://user-images.githubusercontent.com/108667911/182804339-3c3d15b0-afff-4707-83f8-4b22ebf64af4.png">

Reply:
So it's probably complaining about I18n.t("#{searched_term}.title")

Reply:
****
```suggestion
      expect(searched_term).to eq(I18n.t("#{searched_term}.title"))
```